Assets/Scripts/SethRefactor/ActionLogics/Move.cs

```diff
@@ -7,24 +7,31 @@ namespace ScavengerWorld
     [CreateAssetMenu(menuName = "Scavenger World/Action Logics/Move")]
     public class Move : ActionLogic
     {
-        public override bool RequiresInRange(Unit agent, Interactable target)
+        public override bool RequiresInRange(Unit unit, Interactable target)
         {
             return false;
         }
 
-        public override void StartAction(Unit agent, Interactable target)
+        public override void StartAction(Unit unit, Interactable target)
```

Comment:
Why does moving require an interactable target?  Why not just a location?

Reply:
Oh it looks like this particular action is for moving to an interactable, whereas the `Mover` class can just call go to point on the navmesh.  Got it.

Reply:
Yeah this is a hacky solution for now. Ideally we would  not have a Move action at all.